cores/esp32/esp32-hal-gpio.h

```diff
@@ -42,7 +42,7 @@ extern "C" {
 
 //GPIO FUNCTIONS
 #define INPUT             0x01
-#define OUTPUT            0x02
+#define OUTPUT            0x03 //Changed from 0x02 to behave the same as Arduino pinMode(pin,OUTPUT) where yo ucan read the state of pin even when it is set as OUTPUT
```

Comment:
`where yo ucan` typo... It may also make sense to put the comment on it's own line.

Reply:
Thanks for pointing that out. Fixed :)